### ISSUE TITLE
Notifications don't disappear and the close button does not work

### ISSUE THREAD

**Issue by ArcTheSpark2** [Leantime/leantime#1592]
**Describe the bug**
When I notification opens, It stays on the page and does not disappear after a few seconds like the older versions and trying to manually close it does not work either

**To Reproduce**
1. Click on the "Meatball" menu on a task
2. Click on the "Start Work" button or "Stop Work" button
3. The notification pop up on the top right of the screen and stays there.

**Expected behavior**
The notification should disappear after a few seconds and the close button should close the notification of clicked.

**Leantime Version**
2.3.20

**Server**
I'm using the Docker image "leantime/leantime"
https://hub.docker.com/r/leantime/leantime

**PHP / MySQL Version**
MySQL 8.0

**Additional context**
I used Leantime Version 2.2.8 for a long time and it worked 100% there. I upgraded to 2.3.14, That was when this issue / bug started. Today I upgraded to version 2.3.20 and the issue persists.

I have tried on multiple OS's (MacOs, ubuntu and windows) using firefox, safari and Edge. The issue is the same.

Here is the docker-compose I use:

```
version: '3.3'

services:
  leantime_db:
    image: mysql:8.0
    container_name: mysql_leantime
    volumes:
      - /home/username/docker/leantime/db_data:/var/lib/mysql
    restart: unless-stopped
    environment:
      MYSQL_ROOT_PASSWORD: 'coolPassword'
      MYSQL_DATABASE: 'leantime'
      MYSQL_USER: 'leantimeuser'
      MYSQL_PASSWORD: 'coolPassword'
    command: --character-set-server=utf8 --collation-server=utf8_unicode_ci

  leantime:
    image: leantime/leantime:latest
    container_name: leantime
    restart: unless-stopped
    environment:
      # LEAN_APP_URL: 'https://domain.com/leantime'                     # Only needed for subdirectory setup; protocol (http or https) and base URL , trailing slash not needed
      LEAN_SITENAME: 'Leantime'                                         # Name of your site, can be changed later
      LEAN_DB_HOST: 'mysql_leantime'                                    # Database host, derived from container_name in leantime_db container
      LEAN_DB_USER: 'leantimeuser'
      LEAN_DB_PASSWORD: 'coolPassword'
      LEAN_DB_DATABASE: 'leantime'
      LEAN_DEFAULT_TIMEZONE: 'Africa/Johannesburg'                         # Set default server timezone
      LEAN_SESSION_PASSWORD: 'coolPassword'         # Salting sessions. Replace with a strong password
      LEAN_SESSION_EXPIRATION: 28800                                    # How many seconds after inactivity should we logout?  28800seconds = 8hours
      PUID: 1000
      PGID: 1000
    volumes:
      - /home/username/docker/leantime/public_userfiles:/var/www/html/public/userfiles
      - /home/username/docker/leantime/userfiles:/var/www/html/userfiles
    ports:
      - "83:80"                                                       # The port to expose and access Leantime
    depends_on:
      - leantime_db                                                     # Don't start Leantime unles leantime_db is running

  leantime-db-backup:
    container_name: leantime-db-backup
    image: tiredofit/db-backup
    links:
     - leantime_db
    volumes:
      - /home/username/docker/leantime/backups:/backup
    environment:
      # - DEBUG_MODE=TRUE
      DB_TYPE: 'mysql'
      DB_HOST: 'leantime_db'
      DB_NAME: 'leantime'
      DB_USER: 'leantimeuser'
      DB_PASS: 'coolPassword'
      DB_DUMP_FREQ: 1440           # backup every two hours
      DB_DUMP_BEGIN: 0000      # backup starts immediately
      DB_CLEANUP_TIME: 2880         # clean backups they are older than 2 days
      CHECKSUM: 'SHA1'
      COMPRESSION: 'GZ'
      SPLIT_DB: 'FALSE'
      CONTAINER_ENABLE_MONITORING: 'FALSE'
    restart: unless-stopped
```

**Comment by marcelfolaron** [comment]
Thanks for the detailed write up. I will take a look and submit a bug fix

**Comment by marcelfolaron** [comment]
Fixed in 2.3.21

**Comment by ArcTheSpark2** [comment]
thank you very much for your awesome work.
